mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -479,6 +500,25 @@ object KMeans {
   }
 
   /**
+   * Trains a k-means model using the given set of parameters and initial cluster centers
+   *
+   * @param data training points stored as `RDD[Vector]`
+   * @param k number of clusters
+   * @param maxIterations max number of iterations
+   * @param initialModel an existing set of cluster centers.
+   */
+  def train(
```

Comment:
I'm not sure at this point what the thinking is on adding yet another overload to the utility method. At some point one is expected to use `KMeans` directly, and I recall some move to stop adding these utility methods. But I am not sure -- @mengxr @jkbradley any opinion?